libzmix/src/signatures/delg_cred_cdd/errors.rs

```diff
@@ -0,0 +1,135 @@
+use failure::Error;
+
+#[derive(Debug, Fail)]
+pub enum DelgError {
```

Comment:
Can this be renamed to `DelegationCredentialError` or some full text variant?

Reply:
Changed to more appropriate name